worker/src/main/java/gov/cms/ab2d/worker/processor/domainmodel/ProgressTracker.java

```diff
@@ -57,5 +64,9 @@ public int getPercentageCompleted() {
         return percentCompleted;
     }
 
+    public boolean isErrorCountBelowThreshold() {
+        return (failureCount * 100) / processedCount < failureThreshold;
```

Comment:
The failure threshold should be calculated against the total number of patients in the contract.

Reply:
done.